modules/admin/client/components/Admin.component.js

```diff
@@ -0,0 +1,28 @@
+import React from 'react';
+
+export default function Admin() {
+  return (
+    <>
+      <style>
+        {`#wrapper {
+          margin-top: 60px;
+        }`}
+      </style>
```

Comment:
You can use `container-spacer` class instead:
```html
<div className="container container-spacer">
```

In general, any CSS should go on its own stylesheet.

Let's avoid using `id` selectors, especially generic ones like this as they're difficult to manage and harm re-usability. It's okay to create specific class-selectors instead.

Reply:
This was a dirty solution i added to see a _Hello admin_ hidden behind a header. :)

Reply:
No worries. 🙂  `container-spacer` for the win! 💪